### PR TITLE
Lang v062

### PR DESCRIPTION
**Pull request by emmamarichal**
solves Chinese specimen issue

---

b/343275802

### CODE REVIEW

File: lang/Lib/gflanguages/data/languages/gan_Hant.textproto [hunk 1]
```diff
@@ -2,8 +2,13 @@ id: "gan_Hant"
 language: "gan"
 script: "Hant"
 name: "Gan Chinese"
+<<<<<<< HEAD
 preferred_name: "Gan"
 autonym: "Gan Chinese（Traditional）"
+=======
+preferred_name: "Gan Chinese"
+autonym: "贛語"
+>>>>>>> f6a234f61ded3f2c386870192ad701f47105c23b
```
Comment (nathan-williams) [inline lang/Lib/gflanguages/data/languages/gan_Hant.textproto:11]: Looks like a bad merge. Please fix.

File: lang/Lib/gflanguages/data/languages/hsn_Hant.textproto [hunk 2]
```diff
@@ -2,7 +2,11 @@ id: "hsn_Hant"
 language: "hsn"
 script: "Hant"
 name: "Xiang Chinese"
+<<<<<<< HEAD
```
Comment (nathan-williams) [inline lang/Lib/gflanguages/data/languages/hsn_Hant.textproto:5]: Please resolve merge diffs.

File: lang/Lib/gflanguages/data/languages/wuu_Hant.textproto [hunk 3]
```diff
@@ -2,7 +2,12 @@ id: "wuu_Hant"
 language: "wuu"
 script: "Hant"
 name: "Wu Chinese"
+<<<<<<< HEAD
```
Comment (nathan-williams) [inline lang/Lib/gflanguages/data/languages/wuu_Hant.textproto:5]: ""

**Review by nathan-williams — CHANGES_REQUESTED** [review]

**Comment by emmamarichal** [comment]
@nathan-williams, done! you can merge it. (It seems I can't merge it myself)

**Review by m4rc1e — APPROVED** [review]

**Pull request merged**